### ISSUE TITLE
\Description

### ISSUE THREAD

**Issue by mimbres** [arXiv/html_feedback#6692]
### Description

\Description is broken

### (Optional:) Please add any files, screenshots, or other information here.

https://arxiv.org/html/2502.21267v1 section 2.3

### (Required) What is this issue most closely related to? Select one.

Other

### Internal issue ID

d5e87da0-7596-42d0-9f5e-cbf73e83b0bc

### Paper URL

https://arxiv.org/html/2502.21267v1

### Browser

Chrome/149.0.0.0

### Device Type

Desktop

**Comment by dginev** [comment]
Thank you for your report - the specific issue is now solved.

We have regenerated the article and you should be able to see an improvement.

Feel free to open new tickets for any remaining issues.
